### DISCUSSION TITLE
Dark mode prevent.

### DISCUSSION THREAD

**Discussion by yoann54** [maizzle/framework#1364] [General]
Is there a way to prevent dark mode to happen on the resulting email ?

**Comment by cossssmin** [comment]
It depends, see here for an in-depth article:

https://www.litmus.com/blog/the-ultimate-guide-to-dark-mode-for-email-marketers

**Reply by yoann54** [reply]
Thanks for your answer im digging into the article right now. It seems i have a problem with the native dark mode, i have to put many overwriting rules in the reset.css
